pagesections/main.statement.section.js

```diff
@@ -60,7 +60,7 @@ const MainStatementSection = ( Base ) => class extends Base {
 
 		this.mainStatementsContainer.$(
 			this.constructor.STATEMENT_WIDGET_SELECTORS.EDIT_INPUT_VALUE
-		).waitForExist( null, true );
+		).waitForExist( true );
```

Comment:
I believe the intention of this line is to be sure that the edit box has disappeared. The idea behind this is that would be a proxy for the edit having been finished and saved.

I believe this exact line was written while we were still on wdio 3 or before (!) at which point the params meant: `timeout` and `reverse`. The idea was that we didn't want to alter the timeout (hence the `null`) but we did want to `reverse` the test for existence.

Given this I think the parameter should instead probably be `{reverse: true}` rather than plain true